public/firebase-messaging-sw.js

```diff
@@ -0,0 +1,31 @@
+// eslint-disable-next-line no-undef
+importScripts("https://www.gstatic.com/firebasejs/8.8.0/firebase-app.js");
+// eslint-disable-next-line no-undef
+importScripts("https://www.gstatic.com/firebasejs/8.8.0/firebase-messaging.js");
```

Comment:
I see that you use namespace api here. Could you try to use modular API as described [here](https://firebase.google.com/docs/cloud-messaging/js/receive#web-modular-api_2)?